Parser.fs

```diff
@@ -193,10 +210,9 @@ let parseBinaryExpressionLevel nextLevel expList =
         // TODO(CaptainHayashi): can this be solved without backtracking?
         nodify (pstring ops)
         .>>? ws
-        (* A binary operator cannot ever be followed by a semicolon, + or -.
-           This check removes ambiguity between > and <atomic braces>;,
-           + and ++, and - and --. *)
-        .>>? notFollowedBy (anyOf ";+-")
+        (* A binary operator cannot ever be followed by + or -.
+           This check removes ambiguity between + and ++, and - and --. *)
+        .>>? notFollowedBy (anyOf "+-")
```

Comment:
Changing from `< >` to `<| |>` means we lose the parser ambiguity formerly induced by `< foo = x > 3 >;`, so we can simplify this.